frontend/src/routes/Governance/policies/policy-details/PolicyTemplateDetails.tsx

```diff
@@ -346,13 +368,97 @@ export function PolicyTemplateDetails() {
     [t, kind]
   )
 
+  const relatedResourceFromSearchAPIColumns = useMemo(
+    () => [
+      {
+        header: t('Name'),
+        cell: 'name',
+        sort: 'name',
+        search: 'name',
+      },
+      {
+        header: t('Namespace'),
+        cell: (item: any) => item.namespace ?? '-',
+        search: (item: any) => item.namespace,
+        sort: (a: any, b: any) => compareStrings(a.namespace, b.namespace),
+      },
+      {
+        header: t('Kind'),
+        cell: 'kind',
+        sort: 'kind',
+        search: 'kind',
+      },
+      {
+        header: t('API groups'),
+        cell: (item: any) => item.apigroup ?? '-',
+        sort: 'apigroup',
+        search: 'apigroup',
+      },
+      {
+        header: '',
+        cell: (item: any) => {
+          let policyReportLink: ReactNode = <></>
+          if (item.policyReport) {
+            const { cluster, kind, name, namespace, apiversion } = item.policyReport
+            const namespaceArg = namespace ? `&namespace=${namespace}` : ''
+            policyReportLink = (
+              <span>
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer"
+                  href={`${NavigationPath.resourceYAML}?cluster=${cluster}&kind=${kind}&apiversion=${apiversion}&&name=${name}${namespaceArg}`}
+                >
+                  {t('View policy report')}{' '}
+                  <ExternalLinkAltIcon style={{ verticalAlign: '-0.125em', marginLeft: '8px' }} />
+                </a>
+              </span>
+            )
+          }
+
+          const { cluster, kind, name, namespace, apiversion } = item
+          const namespaceArg = namespace ? `&namespace=${namespace}` : ''
+          return (
+            <Flex>
+              {policyReportLink}
+              <Divider
```

Comment:
Divider should be in the `policyReportLink` variable.

Reply:
why?

Reply:
We only want the divider if there's a policy report or else it'll look funny.

Reply:
gotcha
Good point!!